lambda/main.go

```diff
@@ -52,25 +55,33 @@ func HandleRequest(ctx context.Context) error {
 	// gather latest exploit info from different sources
 	isComplete, err := commands.FetchAllDB()
 	if err != nil {
+		sendSlackNotification(fmt.Sprintf("[%s] Function Failed", functionName), fmt.Sprintf("Error fetching exploit info: %s", err.Error()), true)
 		return err
 	}
 
 	if !isComplete && s3DownloadFailure {
-		return errors.New("skipping db update due to s3 download failure while updating latest exploit info")
+		message := "Skipping db update due to s3 download failure while updating latest exploit info"
```

Comment:
Skipping db update due to incomplete exploit  information fetched from different data  sources and unavailability of last uploaded db file in S3 (error: ...s3 download error message... from line 50 can be used)